source/Calamari.Tests/KubernetesFixtures/KubernetesRunsScriptFixture.cs

```diff
@@ -26,7 +26,7 @@ namespace Calamari.Tests.KubernetesFixtures
     [TestFixture]
     public class KubernetesContextScriptWrapperFixture
     {
-        const string ServerUrl = "<server>";
+        const string ServerUrl = "https://ThisIsAFakeServerUrlWithTheRegion.gr7.ap-southeast-2.eks.amazonaws.com";
```

Comment:
As a part of this change we've switched to using the `aws eks get-token` from the aws cli. In the context of kubectl config this requires a region argument to make a valid request. I haven't been able to find anything indicating that any EKS clusters wouldn't match [this format](https://docs.aws.amazon.com/general/latest/gr/rande.html). If there are issues with this it'll fall through to `aws-iam-authenticator`